checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -299,6 +302,24 @@ protected AnnotatedTypeFormatter createAnnotatedTypeFormatter() {
                 super.methodFromUse(tree);
         AnnotatedExecutableType method = mfuPair.first;
 
+        // Change the type any uninferred type arguments to @NonNull @Initialized.
```

Comment:
This caused new failures in plume-lib: https://travis-ci.org/smillst/checker-framework/jobs/249713675#L1820.